Packs/Vectra_AI/README.md

```diff
@@ -5,26 +5,30 @@ Learn more at [Vectra Website](https://www.vectra.ai).
 
 This pack is designed to quickly integrate with Vectra Detect platform to detect and analyze malicious attacks in progress by creating incident based on Accounts, Hosts or Detections. It gives security engineers visibility into advanced threats to speed detection and remediation response times.
 
+<~XSIAM>This pack includes Cortex XSIAM content.</~XSIAM>
+
 ## What does this pack do?
 
 * Mirrors incidents between Cortex XSOAR incidents and Vectra Detect Accounts, Hosts and Detections alerts.
 * Enriches incidents
 * Download detection PCAP
 * Push tags to Vectra Detect platform
 * Create/Update/Resolve Vectra assignments
+* Create XSIAM Events from Detections and Audits
 
-## Before you start
-
-Make sure you have the following content packs:
 
-* Base
-* Common Scripts
-* Common Types
+## Configuration on Server Side
 
-## Pack configuration
+To get up and running with this pack, you must have a valid API token on your Vectra AI instance. In your Vectra AI instance:
 
-To get up and running with this pack, you must have a valid API token on your Vectra Detect instance.  
+1. Navigate to *My Profile*.
+2. Click on the *General* tab.
+3. Create an API Token.  
 
-It can be retrieved from the Vectra UI > My Profile > General \(tab\) > API Token.  
 Be sure that the user has a role with sufficient permissions to do all the actions.
 
+<~XSIAM>
```

Comment:
Not sure how this tag works, but do you need to end it somewhere with </~XSIAM>. Does this even need to be tagged for XSIAM if it is an XSIAM pack?

Reply:
Good catch, I forgot to add the closing tag. Adding the closing tag will ensure that this section will be hidden in XSOAR.